include/exadg/aero_acoustic/user_interface/field_functions.h

```diff
@@ -47,6 +47,24 @@ struct FieldFunctions
    * space for a given time.
    */
   std::shared_ptr<Utilities::SpatialAwareFunction<dim>> source_term_blend_in;
+
+  /*
+   * The analytical solution of the CFD velocity, that can be used to compute
+   * the aero acoustic source term.
+   */
+  std::shared_ptr<dealii::Function<dim>> analytical_cfd_solution_velocity;
+
+  /*
+   * The analytical solution of the CFD pressure, that can be used to compute
+   * the aero acoustic source term.
+   */
+  std::shared_ptr<dealii::Function<dim>> analytical_cfd_solution_pressure;
```

Comment:
This is a duplication with `IncNS::FieldFunctions`, see comment https://github.com/exadg/exadg/pull/659/files#r1599659156.

Reply:
[b7860ce](https://github.com/exadg/exadg/pull/659/commits/b7860ce30c3cbaeee80a8cfc4f1a4c3247d59845)